components/gly_gen/glycosight/index.tsx

```diff
@@ -0,0 +1,149 @@
+              >
+                {/* <Column
+                  name="Index"
+                  cellRenderer={row => <Cell key={row+''}>{gs_items[row].key.toString()}</Cell>}
+                /> */}
+                <Column
+                  name="UniProt Accession"
+                  cellRenderer={row => <Cell key={row+''}>{gs_items[row].UniProtAcc}</Cell>}
+                />
+                <Column
+                  name="Amino Acid Position"
+                  cellRenderer={row => <Cell key={row+''}>{gs_items[row].AAPosition}</Cell>}
+                />
+                <Column
+                  name="Spectral Count"
+                  cellRenderer={row => <Cell key={row+''}>{gs_items[row].SpectralCount}</Cell>}
+                />
+                <Column
+                  name="Distinct Peptide Count"
+                  cellRenderer={row => <Cell key={row+''}>{gs_items[row].DistinctPeptideCount}</Cell>}
+                />
+                <Column
+                  name="Peptides"
+                  cellRenderer={row => <Cell key={row+''}>{gs_items[row].Peptides}</Cell>}
+                />
+              </Table>
+            )
+          })
+    .build()
+
+export const GlycoSightFileURLNode = MetaNode("GlycoSightURL")
+    .meta({
+        label:"GlycoSight File URL",
+        description: "URL to storage of MZID archives",
+        icon: [glygen_icon],
+    })
+    .codec(GlycoSightFileURLC)
+    .view((data) => (
+        <div>
+            <h2 className="prose max-w-none">GlycoSight File: {data.url}</h2>
+        </div>
+    ))
+    .build()
+
+export const GlycoSightProcessNode = MetaNode("GlycoSightProcessNode")
+    .meta({
+        label: `Launch GlycoSight Analysis`,
+        description: `Run GlycoSight Analysis on MZID file(s)`,
+        icon: [glygen_icon],
+    })
+    .inputs({ file: GlycoSightFileURLNode })
+    .output(GlycoSightOutputNode)
+    .resolve(( props ) => {
+        const result = UploadAndAnalyze(props.inputs.file);
+        return result;
+    })
+    .story((props) => {return "The N-Linked glycan sites analysis was launched through the GlycoSight servers"})
+    .build()
+
+export const GlycoSightFileUpload = MetaNode("GlycoSightUpload")
+    .meta({
+        label: "Upload GlycoSight MZID",
+        description: "Upload MZID data for peptide-to-Uniprot sample mapping",
+        icon: [glygen_icon],
+    })
+    .codec(FileC)
+    .inputs() 
+    .output(GlycoSightFileURLNode)
+    .prompt(props => <>
+        <FilePrompt {...props} example={clientLoadExample} />
+        {props.output ? 
+            <SafeRender 
+                component={FileURL.view} 
+                props={props.output} 
+            /> : 
+            null}</>
+        )
+        .resolve(
+            async (props) => await python(
+                'components.gly_gen.glycosight.load_mzid_file',
+                { kargs: [props.data] },
+                message => props.notify({ type: 'info', message })
+            )
```

Comment:
Since this does nothing -- there is no need to have any python code.. it can just be:
`async (props) => props.data)`
furthermore, prompts will do this by default so resolve can just be omitted altogether.

Reply:
Thanks. This wasn't immediately clear so I went with a no-op.